spark-operator-api/src/test/java/org/apache/spark/k8s/operator/spec/RestartPolicyTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.spark.k8s.operator.spec;
+
+import static org.apache.spark.k8s.operator.spec.RestartPolicy.OnFailure;
+import static org.apache.spark.k8s.operator.spec.RestartPolicy.OnInfrastructureFailure;
+import static org.apache.spark.k8s.operator.spec.RestartPolicy.attemptRestartOnState;
+import static org.apache.spark.k8s.operator.status.ApplicationStateSummary.DriverEvicted;
+import static org.apache.spark.k8s.operator.status.ApplicationStateSummary.DriverReadyTimedOut;
+import static org.apache.spark.k8s.operator.status.ApplicationStateSummary.DriverStartTimedOut;
+import static org.apache.spark.k8s.operator.status.ApplicationStateSummary.ExecutorsStartTimedOut;
+import static org.apache.spark.k8s.operator.status.ApplicationStateSummary.Failed;
+import static org.apache.spark.k8s.operator.status.ApplicationStateSummary.Succeeded;
```

Comment:
I suggested to import `*`, but is this a result of one of `StaticAnalyzers`?

Reply:
it's actually the Java style guide that discourages wildcard  - https://github.com/apache/spark-kubernetes-operator/pull/8#discussion_r1580245635 - spotless therefore marks it as violation.